src/Tvheadend.cpp

```diff
@@ -671,6 +679,64 @@ PVR_ERROR CTvheadend::UpdateTimer ( const PVR_TIMER &timer )
   return SendDvrUpdate(m);
 }
 
+PVR_ERROR CTvheadend::AddTimerecording ( const PVR_TIMER &timer )
+{
+  uint32_t u32;
+  dvr_prio_t prio;
+
+  /* Build message */
+  htsmsg_t *m = htsmsg_create_map();
+  htsmsg_add_u32(m, "daysOfWeek",   timer.iWeekdays);
+  htsmsg_add_str(m, "title",        timer.strTitle);
+  htsmsg_add_str(m, "name",         timer.strTitle);
+  htsmsg_add_u32(m, "channelId",    timer.iClientChannelUid);
+  htsmsg_add_str(m, "description",  timer.strSummary);
+  htsmsg_add_str(m, "comment",      "Created by Kodi Media Center");
+
+  /* Convert start and stop time to time after midnight */
+  struct tm *tmi;
+  tmi=localtime(&timer.startTime);
+  htsmsg_add_s64(m, "start",        (tmi->tm_hour*60 + tmi->tm_min));
```

Comment:
Yes, it makes sense to use u32 then.
Conversion from s64 to u32 works, tested with tvheadend master.